waspc/src/Wasp/Analyzer/StdTypeDefinitions.hs

```diff
@@ -32,6 +31,7 @@ makeEnumType ''JobExecutor
 makeDeclType ''Job
 makeEnumType ''HttpMethod
 makeDeclType ''Api
+makeDeclType ''App
```

Comment:
Was there a reason to swap this ordering?

Reply:
Since it's template Haskell it expected `Route` to be defined before `App` because `App` now uses `Ref Route` for the email auth definition, something @Martinsos helped me with.